doc/tutorials/using_hybrid_mode_on_nuc.rst

```diff
@@ -32,10 +32,10 @@ as shown in :numref:`hybrid_scenario_on_nuc`.
 Set-up base installation
 ************************
 
-- Use the `Intel NUC Kit NUC11TNBi5 <https://ark.intel.com/content/www/us/en/ark/products/205596/intel-nuc-11-pro-board-nuc11tnbi5.html>`_.
+- Use the `Intel Vecow Kit Vecow SPC-7100 <https://marketplace.intel.com/s/offering/a5b3b000000PReMAAW/vecow-spc7100-series-11th-gen-intel-core-i7i5i3-processor-ultracompact-f>`_.
 - Connect to the serial port as described in :ref:`Connecting to the serial port <connect_serial_port>`.
-- Install Ubuntu 18.04 on your SATA device or on the NVME disk of your
-  Intel NUC.
+- Install Ubuntu 20.04 on your SATA device or on the NVME disk of your
+  Intel Vecow.
 
 .. rst-class:: numbered-step
```

Comment:
Have you validated the Zephyr build instructions referenced here?

Reply:
update